### ISSUE TITLE
Request - Option to have bottom screen still be on 3DS

### ISSUE THREAD

**Issue by Pj7oN6** [PabloMK7/ArticBaseServer#49]
I'm not sure if this is possible, but I thought still being able to use the bottom screen on the 3DS when using Artic Controller would be good feature to add as an option. Like having the top screen run from the emulator while the bottom remains on the 3DS, similar to a Wii U set up.

**Comment by Walegy** [comment]
I agree, besides using my mouse just to use the bottom screen sucks... This would be a lot easier for Yo-kai Watch 3, Ocarina Of Time 3D, and so forth.

**Comment by Linkdash2019** [comment]
The only way I can think of this happening is with streaming the Citra screen to the 3ds but that may be too slow for the 3ds to do... (Sending game data and sending input data is already a lot) maybe "newer" consoles can do it?

**Comment by Pacomatic1** [comment]
We could send a low-res and compressed version for now since this is mainly to help with co-ordination.

**Comment by Scraft161** [comment]
TBH, I think for a lot of games just sending a screenshot every couple of seconds would suffice, and for some just the ability to set a static screenshot once would also do the trick.
right now with games that need touchscreen interactions it is kind of just guessing where things are and any ability to set or send images would be a good aid.

**Comment by dcuevasa** [comment]
The **option** to enable this would be amazing
Even if it is only a screenshot every few seconds (or only while the user touches the screen), it would be the definitive way to play!

**Comment by Iltotore** [comment]
> I agree, besides using my mouse just to use the bottom screen sucks... This would be a lot easier for Yo-kai Watch 3, Ocarina Of Time 3D, and so forth.

This would also be very useful for Pr. Layton games. I use arctic on my old 3DS to stream the game with friends and solve riddles together but the controls with the mouse is not really great and the Arctic Controller mode is not really usable IMO because I don't know where I'm clicking.

> The **option** to enable this would be amazing Even if it is only a screenshot every few seconds (or only while the user touches the screen), it would be the definitive way to play!

Clearly this would already be better for such games.
